internal/pki.go

```diff
@@ -0,0 +1,64 @@
+package internal
```

Comment:
I think it might be better to instead make this live under `internal/pki` and call it something like `package pki`. We did something similar for packages like `internal/consts` and `internal/identity/entity`. Importing might make things more explictly clear as well with things like `pki.ReadPolicyIdentifierBlocks`

Reply:
:+1: makes sense